Task2_3/Lecture.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Task2_3
+{
```

Comment:
Think about create base class for all entities to store Description

Reply:
Request completed. Waiting for review